src/main/resources/lib/repo/query.ts

```diff
@@ -57,13 +56,15 @@ export enum Events {
 }
 
 export function logDataQueryEvent(queryId: string, status: QueryStatus): QueryInfoNode | undefined {
```

Comment:
Is the return value from `logDataQueryEvent` used anywhere ? 
If not, maybe make this void and do away with having to `return` anything at all ? (and eliminate the `else` block)

Reply:
removed return values